src/content/upgrades/beacon-chain/index.md

```diff
@@ -7,51 +7,43 @@ sidebar: true
 image: ../../../assets/upgrades/core.png
 summaryPoint1: The Beacon Chain doesn't change anything about the Ethereum we use today.
 summaryPoint2: It introduced proof-of-stake to the Ethereum ecosystem.
-summaryPoint3: It will coordinate the network, serving as the consensus layer.
-summaryPoint4: It is an essential precursor to upcoming scaling upgrades, such as sharding.
+summaryPoint3: It was merged with the original Ethereum proof-of-work chain in 2022.
+summaryPoint4: It is now handling all consensus logic and block gossip for Ethereum.
 ---
 
 <UpgradeStatus isShipped dateKey="page-upgrades-beacon-date">
-  The Beacon Chain shipped on December 1, 2020. To learn more, <a href="https://beaconscan.com/">explore the data</a>. If you want to help validate the chain, you can <a href="/staking/">stake your ETH</a>.
+  The Beacon Chain shipped on December 1, 2020. At the time it was a separate chain to Ethereum Mainnet. It ran proof-of-stake but did not handle any transactions. The Beacon Chain was merged with the original Ethereum Mainnet in 2022 to take over the consensus logic and block propagation for Ethereum. Since The Merge, there has only been one, single Ethereum chain.
```

Comment:
comma

```suggestion
  The Beacon Chain shipped on December 1, 2020. At the time, it was a separate chain to Ethereum Mainnet. It ran proof-of-stake but did not handle any transactions. The Beacon Chain was merged with the original Ethereum Mainnet in 2022 to take over the consensus logic and block propagation for Ethereum. Since The Merge, there has only been one, single Ethereum chain.
```

We could also remove some unnecessary words here if we wanted. Feel free to ignore if you disagree.

```suggestion
  The Beacon Chain shipped on December 1, 2020. At the time, it was a separate chain to Ethereum Mainnet. It ran proof-of-stake but did not handle any transactions. The Beacon Chain was merged with the original Ethereum Mainnet in 2022 to take over the consensus logic and block propagation for Ethereum. Since The Merge, there has only been one Ethereum chain.
```